src/windows/models/properties_model.py

```diff
@@ -219,57 +219,55 @@ def remove_keyframe(self, item):
             # Get effect object
             c = Effect.get(id=clip_id)
 
-        if c:
-            # Update clip attribute
-            if property_key in c.data:
-                log.info("remove keyframe: %s" % c.data)
-
-                # Determine type of keyframe (normal or color)
-                keyframe_list = []
-                if property_type == "color":
-                    keyframe_list = [c.data[property_key]["red"], c.data[property_key]["blue"], c.data[property_key]["green"]]
-                else:
-                    keyframe_list = [c.data[property_key]]
+        if c and property_key in c.data:  # Update clip attribute
```

Comment:
I like the change, but are you certain it's correct. Does the code have side-effects?

Reply:
I had to triple-check, but yeah it should be safe. The **entire** rest of the method body is inside the `if`, so none of it will ever be executed if the condition fails.

Reply:
(Applying it would require more than just accepting the suggestion, as the rest of the body would _have_ to be outdented to keep Python from complaining about unexpected indentation / unreachable-code-after-`return`.)

Reply:
Actually, Python wouldn't complain about that at all, it's far too forgiving and would just assume we meant to do that (despite it being an absurd thing to do). Fortunately, Codacy would take issue with it.

Reply:
Ah, right. I looked at the wrong code piece. Thank you for mentioning it; I think I will integrate this in flake8-simplify: https://github.com/MartinThoma/flake8-simplify/issues/8